### PR TITLE
Soap is properly gendered

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
Soap uses "it" instead of "they"

## Why It's Good For The Game
Soap isnt alive :|

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

</details>

## Changelog
:cl:
tweak: soap is properly gendered
/:cl:

### CODE REVIEW

File: code/game/objects/items/clown_items.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 /obj/item/soap
 	name = "soap"
 	desc = "A cheap bar of soap. Doesn't smell."
-	gender = PLURAL
+	gender = NEUTER
```
Comment (itsmeow) [inline code/game/objects/items/clown_items.dm]: Isn't NEUTER the default for `/obj`?

Comment (etherware-novice) [reply code/game/objects/items/clown_items.dm]: well uhh the only var/gender = NEUTER i could find was in datum/sprite_accessory, its probably fine?

**Comment by zeskorion** [comment]
soap can have pronouns if it wants to. could you instead make this assign every bar of soap random pronouns on initialize() ?

**Comment by etherware-novice** [comment]
w- why tho

File: code/game/objects/items/clown_items.dm (hunk 1 above)
Comment (PowerfulBacon) [reply code/game/objects/items/clown_items.dm]: Gender is a byond internal and it defaults to neuter
```suggestion
```

**Comment by zeskorion** [comment]
> w- why tho

why not

File: code/game/objects/items/clown_items.dm (hunk 1 above)
Comment (PowerfulBacon) [inline code/game/objects/items/clown_items.dm]:
```suggestion
```

Comment (PowerfulBacon) [reply code/game/objects/items/clown_items.dm]: damn I can't commit to this pr

**Review by itsmeow — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
They are no more, it is all that remains

**Pull request merged**